tests/Type/Symfony/envelope_all.php

```diff
@@ -0,0 +1,9 @@
+<?php declare(strict_types = 1);
+
+$envelope = new \Symfony\Component\Messenger\Envelope(new stdClass());
+
+$test1 = $envelope->all(\Symfony\Component\Messenger\Stamp\ReceivedStamp::class);
+$test2 = $envelope->all(random_bytes(1));
```

Comment:
any better way to pass a "non-constant-string" in this test?

Reply:
I guess this is fine.